src/utils/src/kernel_version.rs

```diff
@@ -1,147 +0,0 @@
-// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Hmm this looks like it could be useful in the future. I agree we can resurrect it, but how will we remember?

Is there a way to keep it? Will clippy complain this is dead code?

Reply:
I don't think it'd complain, as its marked as `pub` inside of a library crate. Tbh if enough time passes, we'd probably forget about this ever having existed, but I'm just really not a big fan of having dead code lying around :/